scripts/compiler.js

```diff
@@ -19,21 +19,26 @@ const SWC_CONFIG = {
     sourceMaps: false,
 };
 
-const startTime = Date.now();
-const filesToCompile = glob.sync('./src/core/**/*.ts');
+async function BuildTargetResource(name) {
```

Comment:
`BuildTargetResource` should be `buildTargetResource`

It is not a class and not invoked with `new` so common convention is `camelCase`